sympy/stats/rv.py

```diff
@@ -788,7 +788,8 @@ def where(condition, given_condition=None, **kwargs):
     Interval.open(-1, 1)
 
     >>> where(And(D1<=D2 , D2<3))
-    Domain: ((Eq(a, 1)) & (Eq(b, 1))) | ((Eq(a, 1)) & (Eq(b, 2))) | ((Eq(a, 2)) & (Eq(b, 2)))    """
+    Domain: (Eq(a, 1) & Eq(b, 1)) | (Eq(a, 1) & Eq(b, 2)) | (Eq(a, 2) & Eq(b, 2))
```

Comment:
The And's don't actually need parentheses around them, either, but it's perhaps nice to remove ambiguity for the reader who infrequently looks at logical expressions.